Meta/Lagom/CMakeLists.txt

```diff
@@ -525,6 +525,10 @@ if (BUILD_LAGOM)
             lagom_test(${source} LIBS LagomGL)
         endforeach()
 
+        # PDF
+        lagom_test(../../Tests/LibPDF/TestPDF.cpp LIBS LagomPDF
+                   WORKING_DIRECTORY ${SERENITY_PROJECT_ROOT}/Base/home/anon/Documents/pdf/)
```

Comment:
Please use the same pattern as below (`file(GLOB)` + `foreach`)

Reply:
Done